ccf/app/views.py

```diff
@@ -38,6 +39,18 @@ def job_create_view(request):
     return render(request, "job_create.html", context)
 
 
+@login_required
+def jobs_view(request):
+    """Jobs Page
+
+    Returns to the user all of their jobs and their status.
+    """
+    jobs_list_unfin = Job.objects.filter(user=request.user).filter(finished=False)
```

Comment:
somewhat pedantic, but calling this `unfinished_jobs` and `finished_jobs` is the same number of characters, but considerably more readable.

Reply:
Yeah, it's probably a good idea for me to change that, thanks.